### ISSUE TITLE
Known bugs

### ISSUE THREAD

**Issue by Vicen04** [Vicen04/Dw1DataAndPatches#22]
These are caused by memory limitations mostly... So there is no simple fix.

Going with **Machinedramon** to **Bettle Land** and triggering a battle when all 4 digimon in that area are present can crash the game. Try to avoid triggering all 4 of them.

The technique "All Range Beam" can crash the game in battles with 3 digimon, it is more likely for it to crash if you have Machinedramon as a partner, but it can technically happen with any digimon with the right conditions...

**Comment by LinconEduardo** [comment]
same problem with Machinedramon in Mt. Infinity firts room

**Comment by Vicen04** [comment]
Yes, probably anything with a lot of Digimon in that room and getting all of them in battle triggers that...

**Comment by jrd656** [comment]
Just took a kunemon to beetleland. Approached a wild dokunemon. Two tentamon joined the dight and a 4th digimon declined. The game didn't crash, but once the two tentamon died, I noticed that I couldn't apply any damage to the remaining dokunemon. Techniques and finishers didn't seem to affect it.

**Comment by Vicen04** [comment]
Did you flatten one of them and then died?
The game has a bug where if you die and then revive while the "flatten" animation is playing... The enemy Digimon will become fully untouchable. 
I fixed it in 2.2 part 1 but I'm not sure which version you're playing.

**Comment by jrd656** [comment]
@Vicen04 ah that is probably what happened. Those tentamon are brutal with their bug technique and I think I had to use a couple of restores.

I think I'm on 2.0 or 2.1. If I update to 2.2 can I still use my saves?

Another bug on the one I'm on (which you may well have fixed) is the grey lord's mansion room with 4 computers with items in. I couldn't leave that room. I had to use an autopilot.
